spec/examples/carauctionbids.json

```diff
@@ -0,0 +1,49 @@
+{
+  "id": "handleCarAuctionBid",
+  "version": "1.0",
+  "name": "Car Auction Bidding Workflow",
+  "description": "Store a single bid whole the car auction is active",
+  "start": {
+    "stateName": "StoreCarAuctionBid",
+    "schedule": "2020-03-20T09:00:00Z/2020-03-20T15:00:00Z"
+  },
+  "functions": [
+    {
+      "name": "StoreBidFunction",
+      "operation": "http://myapis.org/carauctionapi.json#storeBid",
+      "type": "rest"
```

Comment:
This file is slightly different from  https://raw.githubusercontent.com/serverlessworkflow/sdk-java/main/api/src/test/resources/examples/carauctionbids.json, 

I have added the `type": "rest"` property since the sdk is creating objects with default values https://github.com/serverlessworkflow/sdk-typescript/blob/main/src/model/function.builder.ts#L23 


Now I have the doubt: is the sdk responsibility to set default values (as per the specification) or should we leave it blank?

Reply:
yeah you should set the default values if they are not specified. User should not have to set the type here as thats its default.

Reply:
you can do the setting of defaults in another pr if you want, so this one can go in now. up to you

Reply:
I think that my comment is confusing, 

the sdk is doing already the settings of defaults https://github.com/serverlessworkflow/sdk-typescript/blob/main/spec/function-builder.spec.ts#L46 

no need of changes from my side